### PR TITLE
developed main features

### PR DESCRIPTION
**Pull request by CobbyElsonfx**
## In this PR I implemented the following✔:
- Built a metrics web app called BQA which stands for breath quality air. This app basically uses the Openweather api to provide an information about the quality of air in a specific country. 
- Deployed the app using render
- Tested all pure components
- Ensured responsive design on both mobile and desktop

### Project requiremetns :
Click on this Link to access Project Requirements: [Requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)

### Changes requested by Previous Code Reviewer
- Add acknowledgment 
- Add a presentation video that basically highlight some code I am proud of
 - Add a search bar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,131 @@
...
-### `npm run eject`
+🚀 Live Demo
+Link to  deployed project..
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+-[Live Link](https://bqabreatqualityair.onrender.com)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+💽 Live Video
+-[Live Video](https://watch.screencastify.com/v/cxvfVKXkcGGHGgJkc496)
```
Comment (Johnadibe) [inline README.md]: > - [ ] According to the project requirement, when recording a video presentation, you should highlight some interesting piece of code or something you built that you are very proud of.  I've noticed that this aspect was missing from your presentation. I kindly recommend including this as it not only aligns with the project requirements but also adds depth and engagement to your presentation.👍 Additionally, I observed that there was a required feature implementation missing from your presentation, as outlined below👇. Incorporating these aspects will enhance the overall quality of your presentation and fulfill project expectations.
> 
> page
>

File: src/components/Region.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+    </>
+  );
+};
+
+export default Region;
```
Comment (Johnadibe) [inline src/components/Region.jsx]: > - [ ] As per the project requirements, the home page should display a list of categories that can be filtered using certain parameters, such as category name. 
> 
> home
> 
> In your current implementation, it appears that you've created a page with a list of items. While your effort is commendable, there's a slight deviation from the project's intended functionality. To align with the requirements and ensure the project's success, I recommend making the following adjustments:
> 
> Parameter for Categories: Implement a filter or search mechanism that allows users to search for categories based on parameters like category name. This ensures that users can easily find and access the information they need.👍🚀
> 
> page
> 
> Kindly not that a well-structured home page with appropriate filters enhances the user experience. It empowers users to find the information they need quickly, making your web app more user-friendly.😊

You can add the search functionality here:

see

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @CobbyElsonfx👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitFlow correctly 👍🏽
✅ Linter checks passed✔️
✅ The project is a single-page application (SPA) built with React and Redux✔
✅ The data retrieved from the API is stored in the Redux store✔
✅ Every page has a unique route within the SPA✔
✅ The project is deployed and accessible online using Render✔
✅ Created the Home Page
✅ Created the Details Page✔
✅ All unit and integration tests are passing✔
✅ The app is mobile friendly✔

## Required Changes ♻️

> - [ ] I've noticed that your project's README currently lacks an "Acknowledgments" section, as required by the project guidelines. Specifically, this section should give proper credit to Nelson Sakwa, the author of the original design, in accordance with the Creative Commons License. Adding this "Acknowledgments" section is not just a project requirement; it's also an essential practice in open-source and creative projects.👍
> 
> read

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Region.jsx (hunk 2 above)
Comment (CobbyElsonfx) [reply src/components/Region.jsx]: Thanks for reviewing my code, I will implement the suggested changes

**Review by Davidosky007 — APPROVED** [review]
Hi @CobbyElsonfx 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All tests passed 👍 
* Good looking design 👍 
* No linter errors 👍 
* Good video presentation 👍 
* Live demo link works 👍 
* All pages work well 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**